lightly/api/api_workflow_sampling.py

```diff
@@ -55,6 +55,8 @@ def sampling(self, sampler_config: SamplerConfig, al_scores: Dict[str, List[np.n
             if preselected_tag_id is None:
                 raise ValueError
             for score_type, score_values in al_scores.items():
+                if isinstance(score_values, np.ndarray):
+                    score_values = score_values.astype(np.float64)
```

Comment:
As a general question, would it not be more efficient to send float32 instead of float64 to the api?

Reply:
`float32` is not supported by th generated openapi REST client. However, we could do the parsing to a string ourselves and pass a string to be uploaded.

Reply:
ok, then let's go with the current solution